scripts/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-use-before-define */
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const list = document.getElementById('list');
+
+let books = [];
+
+function addingBookToHtml(Title, Author, index) {
+  const div = document.createElement('div');
+  div.className = 'list';
+  div.setAttribute('id', index);
+  const title = document.createElement('p');
+  title.innerHTML = Title;
+  title.className = 'bookTitle';
+  const author = document.createElement('p');
+  author.innerHTML = Author;
+  author.className = 'bookAuthor';
+  const button = document.createElement('button');
+  button.innerHTML = 'Remove';
+  button.className = 'RemoveButton';
+  button.setAttribute('id', 'removeButton');
+
+  button.addEventListener('click', ((e) => {
+    const parent = e.target.parentElement;
+    const index = parent.id;
+    books.splice(index, 1);
+    addAllBooks();
+  }));
+
+  const hr = document.createElement('hr');
+  div.appendChild(title);
+  div.appendChild(author);
+  div.appendChild(button);
+  div.appendChild(hr);
+  list.appendChild(div);
+}
+
+function addAllBooks() {
+  list.innerHTML = '';
+  books.forEach((book, index) => {
+    addingBookToHtml(book.title, book.author, index);
+  });
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  books.push({ title: title.value, author: author.value });
+  addAllBooks();
+  title.value = '';
+  author.value = '';
+  localStorage.books = JSON.stringify(books);
+});
+
+window.addEventListener('load', () => {
```

Comment:
- [OPTIONAL] I suggest that you use the `DOMContentLoaded` event instead of the `load` event because this event is used only to detect a fully-loaded page, which can slow the page